src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php

```diff
@@ -347,7 +355,9 @@ protected function setEmpNumbers(ClaimRequestSearchFilterParams $claimRequestSea
         );
 
         if (!is_null($empNumber)) {
-            if (!$this->getEmployeeService()->getEmployeeDao()->getEmployeeByEmpNumber($empNumber) instanceof Employee) {
+            if (!$this->getEmployeeService()->getEmployeeDao()->getEmployeeByEmpNumber(
+                $empNumber
+            ) instanceof Employee) {
                 throw $this->getRecordNotFoundException();
```

Comment:
seems record not found already handled above :thinking:

Reply:
here it's handling for the getAll query parameter